src/backend/joanie/core/serializers/admin.py

```diff
@@ -26,8 +26,38 @@ class AdminCourseLightSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = models.Course
-        fields = ("code", "title", "id")
-        read_only_fields = ("code", "title", "id")
+        fields = ("code", "title", "id", "state")
+        read_only_fields = ("code", "title", "id", "state")
+
+
+class AdminUserSerializer(serializers.ModelSerializer):
+    """Serializer for User model."""
+
+    full_name = serializers.CharField(source="get_full_name")
+
+    class Meta:
+        model = models.User
+        fields = ["id", "username", "full_name"]
+        read_only_fields = ["id", "username", "full_name"]
```

Comment:
Should `fields` and `read_only_fields` be a tuple type instead of a list ? The class Meta of other Serializers seems to be using a tuple instead of a list type.

Reply:
Yup, you're right. I try to homogeneize this point but I miss something here 😉